### PR TITLE
Dev/anubha/comp upgrade for ft tasks

### PR DESCRIPTION
**Pull request by Anubha98**
Change details
1. transformers, accelerate, mlflow package updates
2. env update for tasks -> chat-completion, question-answering, text-classification
3. adapter script changes to remove copying base model dependencies

### CODE REVIEW

File: assets/training/finetune_acft_hf_nlp/components/preprocess/chat_completion/spec.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+version: 0.0.54
 type: command
 
 is_deterministic: True
 
 display_name: Chat Completion DataPreProcess
 description: Component to preprocess data for chat completion task. See [docs](https://aka.ms/azureml/components/chat_completion_datapreprocess) to learn more.
 
-environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/63
+environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/64
```
Comment (msp8955) [inline assets/training/finetune_acft_hf_nlp/components/preprocess/chat_completion/spec.yaml:11]: Can you update nlp_multiclass nlp_multilabel nlp_ner as well?
those are referring to
environment: azureml://registries/azureml/environments/acft-hf-nlp-gpu/versions/16

Comment (Anubha98) [reply assets/training/finetune_acft_hf_nlp/components/preprocess/chat_completion/spec.yaml:11]: @msp8955,  we are not owner of the components and do not know how to test them

**Review by aggarwal-k — DISMISSED** [review]

File: assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml [hunk 2]
```diff
@@ -669,9 +669,9 @@ jobs:
       model_selector_output: '${{parent.jobs.question_answering_model_import.outputs.output_dir}}'
     outputs:
       pytorch_model_folder: '${{parent.outputs.pytorch_model_folder}}'
-  ft_nlp_model_converter:
+  question_answering_model_converter:
     type: command
-    component: azureml:ft_nlp_model_converter:0.0.53
+    component: azureml:question_answering_model_converter:0.0.54
```
Comment (skanakamedal) [inline assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml:674]: you need to update the model path at evaluation component

Comment (skanakamedal) [inline assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml:674]: you need to update the model path at evaluation component

**Review by skanakamedal — CHANGES_REQUESTED** [review]

**Comment by skanakamedal** [comment]
Add in PR description which tasks you are upgrading environment for and reasons why other tasks are not being upgraded

File: assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml (hunk 2 above)
Comment (Anubha98) [reply assets/training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml:674]: Fixed, Thanks SitaRam

**Comment by Anubha98** [comment]
> Add in PR description which tasks you are upgrading environment for and reasons why other tasks are not being upgraded

Updated

**Review by aggarwal-k — APPROVED** [review]

**Review by skanakamedal — APPROVED** [review]

**Pull request merged**